app/views/publish/published.html.erb

```diff
@@ -44,13 +45,13 @@
         title: t("publish.published.reviewed.title")
       } %>
 
-      <%= render "govuk_publishing_components/components/govspeak" do %>
-        <%= govspeak_to_html t("publish.published.reviewed.body_govspeak", title: @edition.title) %>
+      <%= render_govspeak(t("publish.published.reviewed.body_govspeak", title: @edition.title)) %>
 
-        <%= link_to(strip_scheme_from_url(public_url), public_url,
-                    class: "govuk-link govuk-link--no-visited-state",
-                    data: { gtm: "view-published-edition" }) %>
-      <% end %>
+      <%= render "govuk_publishing_components/components/inset_text", {
```

Comment:
Have you considered moving this into a partial to DRY up the code, since the same code for the inset text is also used on L20-24?

Reply:
Yes, I noticed the repetition. I guess storing it into a variable using a `capture do` block could be a decent option.

Reply:
@brucebolt updated to use a variable for the published edition link